src/main/java/org/amahi/anywhere/activity/NavigationActivity.java

```diff
@@ -145,7 +145,9 @@ private void setUpInjections() {
     private void setUpHomeNavigation() {
         getSupportActionBar().setHomeButtonEnabled(isNavigationDrawerAvailable());
         getSupportActionBar().setDisplayHomeAsUpEnabled(isNavigationDrawerAvailable());
+/*
```

Comment:
Whats the need for commenting instead of removing?
Also if you want to comment use //
Most text editors/ IDE provide commenting/ uncommenting feature using `ctrl` + `/`